src/documentation/text/01_Preface/task-definition.tex

```diff
@@ -0,0 +1,6 @@
+Die vorliegende Seminararbeit thematisiert den Anwendungsfall eines Ticketverkaufs für Veranstaltungen.
+Kunden und Verwalter erhalten über spezielle Clients Zugriff auf eine Verwaltungskomponente, dessen Implementierung auf einem Aktorensystem basiert.
+Hierbei muss die Implementierung des Systems alle Anwendungsfälle erfüllen.
+Dafür müssen zunächst die Anforderungen an das System gesammelt werden, bevor eine Unterteilung in funktionale und nicht-funktionale Anforderungen stattfindet.
```

Comment:
_bevor eine Unterteilung in funktionale und nicht-funktionale Anforderungen_ 
Muss auf neuste Änderung angepasst werden, da wir keine Unterteilung mehr machen:)

Reply:
Ja, ich wollte noch auf die Zustimmung von allen warten aber ich werde es gleich anpassen 👍